client/src/App.js

```diff
@@ -29,15 +30,18 @@ function App() {
           <Route path="/login">
             <Login />
           </Route>
-            <Route path="/settings">
-              <Settings />
-            </Route>
           <Route path="/passwords">
             <Passwords />
           </Route>
           <Route path="/events">
             <Events />
           </Route>
+          <Route path="/organizations">
+            <Organizations />
+          </Route>
+          <Route path="/programDirectors">
+            <ProgramDirectors />
+          </Route>
```

Comment:
I've created separate organizations/programDirectors routes for the different models...